app/networkapi/settings.py

```diff
@@ -73,6 +73,7 @@
     'networkapi.people',
     'networkapi.features',
     'networkapi.news',
+    'networkapi.utility',
```

Comment:
I made this an app so that the template tags (which honestly are generic and shared between all the real apps) I define in it will be discovered by django.